docs-dev/ADVANCED_CONFIG.md

```diff
@@ -0,0 +1,82 @@
+        "host": STRING                   // complete URI of the IPFS node; used with 'remote'
+        "mode": STRING "remote|bundled"  // mode to run IPFS node in
+        "pinning-endpoints": ARRAY       // array of string endpoints for pinning IPFS data
+        "disable-peer-data-sync": false  // CAUTION- setting to true isolates the node
+    },
+    "logger": {
+        "log-directory": STRING          // path on local filesystem when log-to-files is set
+        "log-level": NUMBER              // Log level. 0 is most verbose
+        "log-to-files": true             // Log to files, default is true
+    },
+    "metrics": {
+         "collector-host": STRING               // hostname of OTLP collector when push exporter enabled
+         "metrics-exporter-enabled": BOOLEAN    // whether push exporting is enabled, default false
+         "prometheus-exporter-enabled": BOOLEAN // whether prometheus-compatible endpoint enalbed, default false
+         "prometheus-exporter-port": NUMBER     // port for scraping prometheus metrics if enabled
+    },
+    "network": {
+        "name": STRING         // Network to connect to: mainnet, testnet-clay, or dev-unstable
+        "pubsub-topic": STRING // CAUTION- setting this will disconnect you from the main ceramic network
+    },
+    "node": {
+        "disable-composedb": BOOLEAN  // CAUTION if true, turns off composedb indexing
+        "private-seed-url": STRING    // the private seed for this node, or will be randomly generated
+        "readOnly": false             // run the node in read-only mode
+        "stream-cache-limit": INTEGER // Max number of streams to keep in memory
+        "sync-override": STRING       // CAUTION overrides internal sync setting for stream loads
+    },
+    "state-store": { 
+				"local-directory": STRING // Defaults to $HOME/.ceramic/statestore
+        "mode": "fs|s3",          // volume storage option shared here, fs=>filesystem, s3=>s3 bucket
+        "s3-bucket": STRING       // Name of S3 bucket
+        "s3-endpoint": STRING     // URL of S3 endpoint
+    }
+}
+```
+
+## Using Environment Variables
+
+In some deployment environments, it is desirable to control the configuration of a node from environment variables that can be set on deployment. 
+The following ceramic daemon configuration variables may be overridden in the environment:
+
+| Environment Variable | Config Section | Config Variable |
+|-----------------|-----------------|-----------------|
+| CERAMIC_DISABLE_IPFS_PEER_DATA_SYNC  | ipfs  | disable-peer-data-sync |
+| CERAMIC_INDEXING_DB_URI    | indexing    | db    |
+| CERAMIC_METRICS_EXPORTER_ENABLED  |  metrics   |  metrics-exporter-enabled   |
+| CERAMIC_NODE_PRIVATE_SEED_URL   | node    |  private-seed-url   |
+| CERAMIC_PROMETHEUS_EXPORTER_ENABLED | metrics  | prometheus-exporter-enabled |
+| CERAMIC_PROMETHEUS_EXPORTER_PORT   | metrics    | prometheus-exporter-port    |
+| COLLECTOR_HOSTNAME  | metrics    | collector-host    |
```

Comment:
Just a comment, I'm kind of shocked there are so few, I thought we have an env for each config file element.

Reply:
ha yeah.  but is so